packages/puppeteer-core/src/api/Page.ts

```diff
@@ -345,6 +345,48 @@ export interface ScreencastOptions {
    * @defaultValue `1`
    */
   speed?: number;
+  /**
+   * Specifies the output file format.
+   *
+   * @defaultValue `webm`
+   */
+  format?: 'webm' | 'gif' | 'mp4';
+  /**
+   * Specifies the frame rate in frames per second.
+   *
+   * @defaultValue `30` (`20` for GIF)
+   */
+  fps?: number;
+  /**
```

Comment:
I think some comments are not attached to the correct value, can you please fix?